org.texmacs.TeXmacs.yml

```diff
@@ -0,0 +1,116 @@
+app-id: org.texmacs.TeXmacs
+runtime: org.kde.Platform
+runtime-version: "5.15-23.08"
+sdk: org.kde.Sdk
+command: texmacs
+rename-desktop-file: texmacs.desktop
+rename-icon: texmacs
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --share=ipc
+  - --filesystem=xdg-documents
+  - --persist=.TeXmacs
+  - --env=PATH=/app/texlive/bin/:/app/texlive/bin/x86_64-linux:/app/texlive/bin/aarch64-linux:/app/bin:/usr/bin
+  - --env=ASPELL_CONF=dict-dir /app/share/dicts
+  - --env=LD_LIBRARY_PATH=/app/texlive/lib:/app/texlive/lib/perl5/5.38.2/aarch64-linux/CORE:/app/texlive/lib/perl5/5.38.2/x86_64-linux/CORE
+  - --env=PERL5LIB=/app/texlive/lib/perl5/5.38.2:/app/texlive/lib/perl5/site_perl/5.38.2
+
+add-extensions:
+  org.freedesktop.Sdk.Extension.texlive:
+    version: "23.08"
+    directory: texlive
+    autodelete: false
+  org.lyx.LyX.Plugin:
+    versions: stable
+    directory: plugins
+    subdirectories: true
+    no-autodownload: true
+    autodelete: true
+    add-ld-path: lib
+
+modules:
+
+  - pythonSoft.yml # symPy, numPy
+
+  - name: texliveYextensions
+    buildsystem: simple
+    build-commands:
+      - install -d /app/texlive
+      - install -d /app/plugins
+
+  - name: dictEs-aspell
+    buildsystem: simple
+    build-commands:
+      - ./configure
+      - make dictdir=/app/share/dicts datadir=/app/share/datadicts
+      - make dictdir=/app/share/dicts datadir=/app/share/datadicts install
```

Comment:
Use autotools and `make-args` `make-install-args`.

Reply:
In this case "--prefix=.." is injected, which causes the configuration script to fail.

Reply:
Weird.